hack/cmd_util.sh

```diff
@@ -174,7 +174,8 @@ function os::cmd::internal::expect_exit_code_run_grep() {
 	os::test::junit::declare_test_start
 
 	local name=$(os::cmd::internal::describe_call "${cmd}" "${cmd_eval_func}" "${grep_args}" "${test_eval_func}")
-	echo "Running ${name}..."
+    local preamble="Running ${name}..."
```

Comment:
Should never have been tabs to begin with (I'll take the hit for it). In some alternate universe when the Bash import PR gets merged ... moving this file into `hack/lib` and changing all the tabs to spaces becomes trivial to do. Maybe the shame of how bad this looks will help that happen ...